src/node_crypto.cc

```diff
@@ -609,7 +621,7 @@ void SecureContext::SetCert(const FunctionCallbackInfo<Value>& args) {
   SecureContext* sc = Unwrap<SecureContext>(args.Holder());
 
   if (args.Length() != 1) {
-    return env->ThrowTypeError("Bad parameter");
+    return env->ThrowTypeError("Only mandatory Certificate string is expected");
```

Comment:
`"Certificate string is mandatory"` or maybe "Certificate string expected"?  The current message reads a little odd.